files/jquery.lazy/update.json

```diff
@@ -3,6 +3,9 @@
   "name": "jquery.lazy",
   "repo": "eisbehr-/jquery.lazy",
   "files": {
-    "include": ["jquery.lazy.min.js"]
+    "include": [
+      "jquery*.min.js",
+      "plugins/jquery*.min.js"
```

Comment:
Ok. I will create a new PR tomorrow. Use this even for the first line? Because in the old file there was no leading ./ ...